### PR TITLE
add sundararajan's results

### PR DESCRIPTION
**Pull request by sundararajan20**
Keeping efficiency of filtering in mind I applied the filters in the following order:
* Gopher quality filters (filters out 73% of the documents)
* Language ID detection (allow only 'en' with a confidence score of 0.9 and above) (filters out 20% of the documents)
* NSFW and Toxic speech detection (allow 'non-nsfw' and 'non-toxic' with confidence scores of 0.9 and above)
* Fasttext quality classifier (allow 'wiki' with a confidence of 0.9 and above) (filters out 4% of the documents)
* Exact line deduplication across all the documents

Filtering took about 11 hours (which the a4-cpu partition allowed up to 64 CPUs) and resulted in a small but high quality dataset of size 1.6B tokens.

The training process samples 26.2B tokens which amounts to 16 epochs through the data I created. I observed no signs of overfitting in terms of the training loss going down but the validation loss creeping up.

Final validation loss: 3.511
Wandb link: https://api.wandb.ai/links/sundararajan-team/brohdggp

### CODE REVIEW

**Comment by nelson-liu** [comment]
thanks!
